MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,569 @@
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings CropTypes="Circle,Square,Free" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.CropTypes = ImageCropType.Square;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize default Effects option
+
+Customize a effect types using `EffectTypes` property in `ImageEditorToolbarSettings` class.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings EffectTypes="Hue, Blur" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.EffectTypes = ImageEffect.Blur;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize default shape option
+
+Customize a shape types using `EffectTypes` property in `ImageEditorToolbarSettings` class.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings Shapes="Circle, Line" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.Shapes = AnnotationShape.Line;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## To customize the ColorPalette
+
+You can change the default colors of the `ColorPalette` in toolbar.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings>
+            <imageEditor:ImageEditorToolbarSettings.ColorPalette>
+                <Color>Yellow</Color>
+                <Color>Pink</Color>
+                <Color>Violet</Color>
+            </imageEditor:ImageEditorToolbarSettings.ColorPalette>
+        </imageEditor:ImageEditorToolbarSettings>
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image4.png");
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Yellow);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Pink);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Violet);
+
+{% endhighlight %}
+
+{% endtabs %}
```

Comment:
Mention color palette colors are common for Text, Shape and Pen

Reply:
Added a note to mentioned